src/Routes.tsx

```diff
@@ -34,6 +35,10 @@ const WorkRise = lazy(() => import("./intl/ug/WorkRise"));
 const MentorDashboard = lazy(() => import("./mentorship/MentorDashboard"));
 
 export interface ChildProps {
+  // history: any;
```

Comment:
So, we aren't passing any `router` object through props to the Route components? Or is it built in?

Reply:
I guess not! Answered my own question.

Reply:
If I am understand what you are asking, then no, and you can no longer use custom wrapped router components and pass the props that way like it was before. you have to use plain Route thats wrapped in Routes.